root-ruby-style.gemspec

```diff
@@ -10,10 +10,10 @@ Gem::Specification.new do |gem|
   gem.summary       = "Root's Ruby/Rails Style Guide"
 
   gem.add_runtime_dependency "activesupport", ">= 5.0", "< 7.x"
-  gem.add_runtime_dependency "rubocop", "~> 0.74.0"
```

Comment:
These are required since this gem integrates with rubocop. When eg. `root-server` pulls this in, it can't select a version of rubocop that `root-ruby-style` isn't known to support.

Reply:
Ah, that makes sense.. I can set the updated version number.